### ISSUE TITLE
Fix static files path

### ISSUE THREAD

**Issue by jmjuanes** [jmjuanes/stattic#1]
```
> coverviewer@0.1.0 test /Users/jmjuanes/git/coverviewer-dev
> node dev.js


Welcome to Stattic
Visit us: http://statticjs.github.io

Static server listening on: http://localhost:5001
Static files listening from: /Users/jmjuanes/git/coverviewer-dev/node_modules/coverviewer-build
```

**Comment by jmjuanes** [comment]
Fixed on v0.1.5 ( 9a366f1373ee2fb6223d7dc871cede9bf3340c96 )

Using `process.cwd()` instead of `__dirname`. 

```
* process.cwd() returns directory where command has been executed (not directory of the node package) if it's has not been changed by 'process.chdir' inside of application.
* __filename returns absolute path to file where it is placed.
* __dirname returns absolute path to directory of __filename.
```
(Extracted from http://stackoverflow.com/questions/12238477/determine-command-line-working-directory-when-running-node-bin-script )

**Comment by jmjuanes** [comment]
Fixed on v0.1.5 ( 9a366f1373ee2fb6223d7dc871cede9bf3340c96 )

Using `process.cwd()` instead of `__dirname`. 

```
* process.cwd() returns directory where command has been executed (not directory of the node package) if it's has not been changed by 'process.chdir' inside of application.
* __filename returns absolute path to file where it is placed.
* __dirname returns absolute path to directory of __filename.
```

(Extracted from http://stackoverflow.com/questions/12238477/determine-command-line-working-directory-when-running-node-bin-script )
